test/unit/permutations/test_permutations.cpp

```diff
@@ -19,6 +19,8 @@
 #include "dlaf_test/matrix/util_tile.h"
 #include "dlaf_test/util_types.h"
 
+#include "dlaf/matrix/print_csv.h"
+
```

Comment:
Is it needed?

Reply:
https://github.com/eth-cscs/DLA-Future/pull/564/commits/51ccf2923500485f08269c2a132f13660a99a51a